sumcheck/src/structs.rs

```diff
@@ -12,7 +14,7 @@ use transcript::Challenge;
     deserialize = "E::BaseField: DeserializeOwned"
 ))]
 pub struct IOPProof<E: ExtensionField> {
-    pub point: Vec<E>,
+    pub point: Point<E>,
```

Comment:
I just found `Point<E>` shouldn't be included in `IOPProof<E>`. What do you think about it?

Reply:
yes I agree we should remove it and I will submit another PR to do that since in involved more change outside of this scope